packages/os/send-boot-success.service

```
@@ -0,0 +1,14 @@
+[Unit]
+Description=Send successful boot metrics
+# Metricdog depends on the proxy.env file created by settings-applier.service
+After=settings-applier.service
+Requires=settings-applier.service
```

Seems like we can just treat this the same as the other multi-user services:
```suggestion
After=network-online.target configured.target
Wants=network-online.target configured.target
```